packages/api/src/modules/tmdb/tmdb.resolver.ts

```diff
@@ -54,4 +57,31 @@ export class TMDBResolver {
   public getRecommendedMovies() {
     return this.tmdbService.getRecommended('movie');
   }
+
+  @Query((_returns) => [TMDBSearchResult])
+  public discover(@Args() args: GetDiscoverQueries) {
+    return this.tmdbService.discover(args);
+  }
+
+  @UseInterceptors(
+    makeCacheInterceptor({
+      key: CacheKeys.TMDB_LANGUAGES,
+      ttl: 1000 * 60 * 60 * 60 * 24, // cache for 1 month
+    })
+  )
+  @Query((_returns) => [TMDBLanguagesResult])
+  public getLanguages() {
+    return this.tmdbService.getLanguages();
+  }
+
+  @UseInterceptors(
+    makeCacheInterceptor({
+      key: CacheKeys.TMDB_GENRES,
+      ttl: 1000 * 60 * 60 * 60 * 24, // cache for 1 month
+    })
+  )
```

Comment:
This is only one request to the tmdb api, I don't think we should cache it it's not an extensive call as well.